src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -27,67 +27,115 @@ using namespace winrt::Windows::UI::Core;
 using namespace winrt::Windows::System;
 using namespace winrt::Windows::UI::Xaml::Controls;
 
+static const winrt::hstring launchTag{ L"Launch_Nav" };
+static const winrt::hstring interactionTag{ L"Interaction_Nav" };
+static const winrt::hstring renderingTag{ L"Rendering_Nav" };
+static const winrt::hstring globalProfileTag{ L"GlobalProfile_Nav" };
+static const winrt::hstring addProfileTag{ L"AddProfile" };
+static const winrt::hstring colorSchemesTag{ L"ColorSchemes_Nav" };
+static const winrt::hstring globalAppearanceTag{ L"GlobalAppearance_Nav" };

```

Comment:
plz don't do this

make them `constexpr` and make them `wstring_view`s. Doing it like this requires you to actually construct an hstring immediately when your library is loaded.

Reply:
(which requires calling out to the windows string APIs)

when in doubt, **AVOID** global static initialization